.github/workflows/conformance-datapath.yaml

```diff
@@ -20,7 +20,7 @@ on:
   #   will disappear from the PR checks: please provide a direct link to the
   #   successful workflow run (can be found from Actions tab) in a comment.
   #
-  # pull_request: {}
+  pull_request: {}
```

Comment:
Let's remove this before merge. I've added the dont-merge label. Please remove it once this has been removed as well.